src/preamble.js

```diff
@@ -505,9 +505,6 @@ function preRun() {
 function ensureInitRuntime() {
 #if STACK_OVERFLOW_CHECK
   checkStackCookie();
-#endif
-#if USE_PTHREADS
-  if (ENVIRONMENT_IS_PTHREAD) return; // PThreads reuse the runtime from the main thread.
 #endif
```

Comment:
this is a cleanup to the pthreads startup code - pthreads set `runtimeInitialized` earlier anyhow, so this check is not needed, the if right after us exits for us. removing it makes the code clearer I think.